docs/make.jl

```diff
@@ -162,11 +161,23 @@ makedocs(bib,
 
 @info "Cleaning up temporary .jld2 and .nc output created by doctests or literated examples..."
 
-for file in vcat(glob("docs/*.jld2"),
-                 glob("docs/*.nc"),
-                 glob("docs/build/generated/*.jld2"),
-                 glob("docs/build/generated/*.nc")
-                 )
```

Comment:
Honestly I prefer the old code. It's not as elegant a solution as this PR is proposing, but it's way clearer and more concise. I believe it also achieves the same result, no?

Reply:
Right. The code is only the same under a very specific assumption.